topic-operator/src/main/java/io/strimzi/operator/topic/v2/BatchingTopicController.java

```diff
@@ -787,6 +796,7 @@ private void deleteInternal(List<ReconcilableTopic> batch, boolean onDeletePath)
                 var e = validate(reconcilableTopic);
                 if (e.isRightEqual(true)) {
                     // adminDelete, removeFinalizer, forgetTopic, updateStatus
+                    reconcilableTopic.startTimer(metrics);
```

Comment:
The reason is to avoid iterating on partitionedByManaged twice, which is also not allowed by the Stream API, so I'll have to clone that stream. I move that to the following map, which I think it's a bit more clean. Wdyt?